src/main/java/org/apache/ibatis/builder/xml/XMLIncludeTransformer.java

```diff
@@ -17,12 +17,16 @@
 
 import org.apache.ibatis.builder.IncompleteElementException;
 import org.apache.ibatis.builder.MapperBuilderAssistant;
+import org.apache.ibatis.parsing.GenericTokenParser;
 import org.apache.ibatis.parsing.PropertyParser;
+import org.apache.ibatis.parsing.TokenHandler;
 import org.apache.ibatis.parsing.XNode;
 import org.apache.ibatis.session.Configuration;
 import org.w3c.dom.Node;
 import org.w3c.dom.NodeList;
 
+import java.util.*;
```

Comment:
1) Remove unused imports and 2) import each class explicitly instead of a package.
Please see some other classes for reference.